<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEditProduce">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRecipe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Recipe name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProduce">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Produce</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIngredients">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingredients</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDairy">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dairy</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditDairy">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Meat</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMeat">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Spices</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSpices">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditOther">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Steps</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditSteps">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditNotes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Link</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditLink">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescription">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditDescription">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescription_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cooking time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescription_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Servings:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRecipe">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCookingTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditServings">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
